From 68e704377b1351d7c5d510b9d6c24d64fa86c143 Mon Sep 17 00:00:00 2001
From: Johannes Meyer <johannes@intermodalics.eu>
Date: Tue, 4 Jun 2019 10:51:59 +0200
Subject: [PATCH] Switch to PCL 1.8+

---
 debian/control.em | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/debian/control.em b/debian/control.em
index 7cbf65a..f3f5e89 100644
--- a/debian/control.em
+++ b/debian/control.em
@@ -2,13 +2,15 @@ Source: @(Package)
 Section: misc
 Priority: extra
 Maintainer: @(Maintainer)
-Build-Depends: debhelper (>= @(debhelper_version).0.0), @(', '.join(BuildDepends))
+Build-Depends: debhelper (>= @(debhelper_version).0.0), libpcl1.8-dev, @(', '.join(BuildDepends))
 Homepage: @(Homepage)
 Standards-Version: 3.9.2
 
-Package: @(Package)
+Package: @(Package)-pcl1.8
+Conflicts: @(Package)
+Replaces: @(Package)
 Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}, @(', '.join(Depends))
+Depends: ${shlibs:Depends}, ${misc:Depends}, libpcl1.8-dev, @(', '.join(Depends))
 @[if Conflicts]Conflicts: @(', '.join(Conflicts))@\n@[end if]@
 @[if Replaces]Replaces: @(', '.join(Replaces))@\n@[end if]@
 Description: @(Description)
-- 
2.7.4

